Sources/NIO/GetaddrinfoResolver.swift

```diff
@@ -21,6 +21,8 @@
 /// needed to implement it.
 ///
 /// This resolver is a single-use object: it can only be used to perform a single host resolution.
+import CNIOLinux
```

Comment:
Is this change needed ?

Reply:
The same as previous, ‘freeifadrrs’ for Android defined in CNIOLinux module.